client/src/components/ProfileDetails/About/About.tsx

```diff
@@ -1,15 +1,39 @@
-import { Avatar, Box, Fade, Grid, GridList, GridListTile, Paper, Typography } from '@material-ui/core';
+import { useEffect, useState } from 'react';
+import { Avatar, Button, Box, Fade, Grid, GridList, GridListTile, Paper, Typography } from '@material-ui/core';
+import { Link } from 'react-router-dom';
 import LocationOnIcon from '@material-ui/icons/LocationOn';
 
 import { Profile } from '../../../interface/Profile';
+import { useAuth } from '../../../context/useAuthContext';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { ReviewsApiData } from '../../../interface/ReviewApiData';
+import { getReviews } from '../../../helpers/APICalls/reviews';
 import useStyles from './useStyles';
+import SitterReview from '../Review/Review';
+import CreateReview from '../CreateReview/CreateReview';
 
 export interface Props {
   sitter: Profile | null | undefined;
 }
 
 export default function About({ sitter }: Props): JSX.Element {
+  const [toggle, setToggle] = useState<boolean>(false);
+  const { loggedInUser, sitterReviews, updateReviewsContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
   const classes = useStyles();
+
+  useEffect(() => {
+    if (sitter) {
+      getReviews(sitter._id).then((data: ReviewsApiData) => {
+        if (data.success) {
+          updateReviewsContext(data.success);
+        } else if (data.error) {
+          updateSnackBarMessage('There was an error fetching reviews!');
+        }
+      });
+    }
+  }, []);
+
   return (
     <>
       {sitter && (
```

Comment:
Check is sitter is passed in parent as it'll return empty element if sitter is null or undefined.
Also sitter is needed in this component so `sitter: Profile | null | undefined;` can be changed to `sitter: Profile;` in Props interface

Reply:
Makes sense! I've changed it accordingly.